app.ts

```diff
@@ -0,0 +1,47 @@
+import { ActiveSession } from "./model/auth";
+
+const express = require('express');
+const path = require('path');
+const nunjucks = require('nunjucks');
+const session =require('express-session')
+
+const app = express();
+
+//configure nunjucks
+const appViews = path.join(__dirname, '/views');
+
+const nunjucksConfig = {
+    autoescape: true,
+    coCache: true,
+    express: app
+};
+
+nunjucks.configure(appViews, nunjucksConfig);
+
+//configure Express
+app.set('view engine', 'html');
+
+app.use('/public', express.static(path.join(__dirname, 'public')));
+
+app.use(express.json())
+
+
+app.use(express.urlencoded({ extended: true }));
+
+app.use(session({secret: 'NOT HARDCODED SECRET', cookie: {maxAge: 60000}}));
```

Comment:
Your secret shouldn't be hard coded, it should be passed via an environment variable